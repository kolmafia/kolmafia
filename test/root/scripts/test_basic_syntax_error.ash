print("no semi-colon here")
print();
