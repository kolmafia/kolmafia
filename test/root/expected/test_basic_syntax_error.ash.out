Expected ;, found print (test_basic_syntax_error.ash, line 2, char 1 to char 6)
